bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/RetryingRpcFutureFallback.java

```diff
@@ -60,7 +48,7 @@ public void sleep(long ms) throws InterruptedException {
   @VisibleForTesting
   BackOff currentBackoff;
   @VisibleForTesting
-  Sleeper sleeper = THREAD_SLEEPER;
+  Sleeper sleeper = Sleeper.THREAD_SLEEPER;
```

Comment:
https://github.com/sduskis/cloud-bigtable-client/blob/retry_auth/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/Sleeper.java

I missed adding it earlier.  sorry about that.